script/Ciel/Day 7/07_01_CIEL07_1C.txt

```diff
@@ -56,7 +56,7 @@ The choice of where to go once classes are over.
 -- Page 76, Offset 14987.
 -- 体調不良で休んでたんじゃないのかよ？　シエル先輩からそう聞いてたんですけどぉ」
 // weird how this conversation flows, I wonder why that is...
-I thought you were taking time off for health reasons?
+I thought you were taking time off for health reasons.
```

Comment:
I don't have the game handy but guessing this is question inflection still?

Reply:
After giving it another listen, the question mark does fit but isnt indirect question like this suppose to end with a period?

Reply:
I feel it's pretty driven by intonation. This seems clarifying rather than just statement so I'd put a ?